adding support for weather icon

ashahar opened the pull request:
add some command line switching

Commit:
add cache data wind direction update url fetching

Commit:
remove timestamp

tupton commented:
You changed much more than the weather icon image tag, which is all the description for this pull request mentions.

What's the use case for getting the image tag? Are you using this script to display it somehow?

I would be much more likely to accept a pull request for these individual things if you split them up: ability to get wind direction, ability to get the weather icon, updated url, updated caching, etc.

I'm unlikely to accept changes to defaults, like the output format and the default separator, but if you want to make options for that (e.g. `--short` for your output format) I would consider that, too.

ashahar commented:
Ok Thanks I'll re-fork it and do it in stages.
I'm using it with conky to display weather forecast and Icon on the desktop.

On Tue, Jul 8, 2014 at 4:44 PM, Thomas Upton notifications@github.com
wrote:

> You changed much more than the weather icon image tag?
> 
> What's the use case for getting the image tag? Are you using this script
> to display it somehow?
> 
> I would be much more likely to accept a pull request for these individual
> things if you split them up: ability to get wind direction, ability to get
> the weather icon, updated url, updated caching, etc.
> 
> I'm unlikely to accept changes to defaults, like the output format and the
> default separator, but if you want to make options for that (--short,
> e.g. for your output format) I would consider that, too.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/tupton/python-yahoo-weather/pull/4#issuecomment-48337363
> .
